typing/includemod.ml

```diff
@@ -606,13 +618,13 @@ and signatures  ~in_eq ~loc env ~mark subst sig1 sig2 mod_shape =
      and the coercion to be applied to it. *)
   let rec pair_components subst paired unpaired = function
       [] ->
-        let oks, (shape_map, deep_modifications), errors =
+        let oks, (shape_map, deep_modifications), errors, unknowns =
```

Comment:
What you mean by "unknowns" was unclear to me, but I think this in fact means `rem(ainder)` or `leftovers`, which I would find clearer.

Reply:
I meant that the remaining elements cannot be categorized clearly into the `oks` or `errors` set. `leftovers` is fine with me.